public/javascripts/PSMap/AddCitiesToMap.js

```diff
@@ -10,19 +10,20 @@
 */
 function AddCitiesToMap(map, citiesData, params) {
     const CITIES_LAYER_NAME = 'cities';
+
+    // Colors from: https://github.com/ProjectSidewalk/Design
     const CIRCLE_CONFIG = {
         fillColor: '#78B9AB', // Project Sidewalk green
         fillOpacity: 0.7,
         strokeColor: '#548177',
-        strokeWidth: 2,
+        strokeWidth: 1,
         radius: 6,
         hoverFillColor: '#FBD78B', // Project Sidewalk yellow
-        hoverStrokeColor: '#b8a06b'
+        hoverStrokeColor: '#b8a06b',
+        privateFillColor: '#EB734D', // Project Sidewalk orange
```

Comment:
With the orange, the private deployments kind of feel like they stand out more than the normal ones.
<img width="441" height="278" alt="image" src="https://github.com/user-attachments/assets/6a51679b-c0ea-4c5e-a199-d95245644fbd" />

Thoughts on making the private ones a bit more transparent?
<img width="441" height="278" alt="image" src="https://github.com/user-attachments/assets/8efded62-c56c-4af9-94f7-fae1bbb39a98" />

Reply:
Ah, good point. I guess we could reduce the size of the orange plus increase the translucency to reduce saliency?